native/glib/Makefile

```diff
@@ -1,26 +1,25 @@
 PKG_NAME = glib
-PKG_SHORT_VERS = 2.58
-PKG_VERS = $(PKG_SHORT_VERS).3
+PKG_SHORT_VERS = 2.62
+PKG_VERS = $(PKG_SHORT_VERS).6
 PKG_EXT = tar.xz
 PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
 PKG_DIST_NAME = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
 PKG_DIST_SITE = https://ftp.gnome.org/pub/GNOME/sources/$(PKG_NAME)/$(PKG_SHORT_VERS)
 
-DEPENDS =
+DEPENDS = 
+#DEPENDS = cross/libffi cross/pcre cross/zlib
 
 HOMEPAGE = https://developer.gnome.org/glib/
 COMMENT  = General-purpose utility library
 LICENSE  = GPLv2
 
-GNU_CONFIGURE = 1
+UNSUPPORTED_ARCHS = powerpc ppc824x ppc853x ppc854x
 
-PRE_CONFIGURE_TARGET = glib_pre_configure
+# https://developer.gnome.org/glib/
+# https://developer.gnome.org/glib/stable/glib-building.html
+# 2.62.6, 2.63.3-OK, 2.63.4-ERROR
+CONFIGURE_ARGS += -Dman=false -Dlibmount=false -Ddefault_library=shared -Dselinux=disabled
+# 2.63.5, 2.66.3
+#CONFIGURE_ARGS += -Dman=false -Dlibmount=disabled -Ddefault_library=shared -Dselinux=disabled
```

Comment:
same as for cross/glib

~but, can we remove the whole package native/glib?~

Reply:
indeed.  Kept the `cross/glib` but removing duplicate comments in `native/glib`.